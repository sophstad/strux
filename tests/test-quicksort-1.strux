int main() {
	int[] a = [1, 4, 3, 6, 7, 2, 99, 23, 37];
  	a.quickSort(9);
	int i = 0;
	for (i = 0; i < 9; i = i + 1) {
		print(a[i]);
	}
	return 0;
}
