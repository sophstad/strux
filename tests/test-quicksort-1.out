[1, 2, 3, 4, 6, 7, 23, 37, 99]
